apps/judicial-system/web/src/components/Table/TableInfoContainer/TableInfoContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import { motion } from 'framer-motion'
+
+import { AlertMessage } from '@island.is/island-ui/core'
+
+import * as styles from './TableInfoContainer.css'
+
+interface TableInfoContainerProps {
+  title: string
+  message: string
+}
+
+const TableInfoContainer: React.FC<TableInfoContainerProps> = (props) => {
```

Comment:
I noticed this exact code and the css in a few different places where we display a message if a table is empty, so I decided to make it reusable. We can then use this when refactoring other case tables when we get around to it